troposphere/static/js/stores/ProjectImageStore.js

```diff
@@ -0,0 +1,103 @@
+define(function (require) {
+
+  var Dispatcher = require('dispatchers/Dispatcher'),
+    BaseStore = require('stores/BaseStore'),
+    ProjectImageCollection = require('collections/ProjectImageCollection'),
+    ProjectImageConstants = require('constants/ProjectImageConstants'),
+    ImageCollection = require('collections/ImageCollection'),
+    Image = require('models/Image'),
+    stores = require('stores');
+
+  var _modelsFor = {};
+  var _isFetchingFor = {};
+  var _pendingProjectImages = new ImageCollection();
+
+  function addPending(model) {
+    _pendingProjectImages.add(model);
+  }
+
+  function removePending(model) {
+    _pendingProjectImages.remove(model);
+  }
+
+  var ProjectStore = BaseStore.extend({
```

Comment:
This filename doesn't match the store variable. Is that intentional? @steve-gregory 

It's _internal_ to the module, I understand. But for consistency, maybe rename.